### PR TITLE
warned users about too much memory

### PR DESCRIPTION
**Pull request by annibond**
Added warning about memory use with the formula on how to figure out a good amount for the user's use case.

### CODE REVIEW

File: downstream/modules/platform/ref-automation-hub-requirements.adoc [hunk 1]
```diff
@@ -22,6 +22,11 @@ For capacity based on forks in your configuration, see link:{BaseURL}/red_hat_an
 
 |===
 
+[IMPORTANT]
+====
+Ansible automation execution nodes and automation hub system requirements are different and may not meet your network's needs. The general formula for determining how much memory you need is: Total control capacity = Total Memory in MB / Fork size in MB.
```
Comment (ianf77) [inline downstream/modules/platform/ref-automation-hub-requirements.adoc:27]: I would say 'might not' instead of 'may not' but that''s all

**Review by ianf77 — COMMENTED** [review]
One minor comment

**Comment by ccopelloRH** [comment]
@annibond @rogrange did this get backported to 2.5? I think I see it in main and 2.4.

**Comment by annibond** [comment]
I do not believe it was backported into 2.5. I didn’t know you could backport 2.5 yet. Sent from my iPhoneOn Jul 12, 2024, at 3:30 PM, ccopelloRH ***@***.***> wrote:﻿
@annibond @rogrange did this get backported to 2.5? I think I see it in main and 2.4.

**Comment by rogrange** [comment]
Apologies, I just saw this. No, as far as I know it did not get backported
to 2.5. At least I did not merge this into the 2.5 branch.

--------------------

Robert Grange

Senior Technical Writer, Ansible

Red Hat <https://www.redhat.com>

100 E Davie St.

Raleigh, N. C. 27601

***@***.***
<https://www.redhat.com>

**Pull request merged**